src/pages/resultsView/coExpression/CoExpressionVizUtils.ts

```diff
@@ -73,16 +81,16 @@ export function computePlotData(
         const yDatum = yData[uniqueSampleKey];
         if (xDatum && yDatum) {
             // only add data if data for both axes
-            const xVal = xDatum.value;
-            const yVal = yDatum.value;
+            const xVal = Number(xDatum.value);
```

Comment:
we typically use parseFloat

Reply:
but parseFloat expects to receive a string, and xDatum.value and xDatum.value sometimes they are strings and sometimes, already numbers.

Reply:
don't you know by this point in the code what type of data you're looking at? and can cast as a string if necessary